src/pages/Main/CanvasDiv.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { canvasElement } from "./Canvas";
+import './CanvasDiv.css';
+
+interface CanvasDivProps {
+    
```

Comment:
And change const CanvasDiv: React.FC<CanvasDivProps> = () => { to CanvasDiv: React.FC = () => {

Reply:
removing the const adds a bug: React Hook "React.useRef" is called in function "React.FC" which is neither a React function component or a custom React Hook function

It looks like it needs to be there.

Reply:
oh I meant to keep the const sorry!